frontend/src/components/plantcard/PlantCard.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import monsteraImage from '../../assets/plant-images/3.jpg'
+
+export default function PlantCard({ name, species }) {
+  PlantCard.propTypes = {
+    name: PropTypes.string.isRequired,
+    species: PropTypes.string.isRequired,
+  }
+
+  return (
+    <CardWrapper>
+      <ImgContainer src={monsteraImage} />
+      <PlantName>{name}</PlantName>
+      <PlantSpecies>{species}</PlantSpecies>
+    </CardWrapper>
+  )
+}
+
+const CardWrapper = styled.div`
+  display: grid;
+  justify-items: center;
+`
+
+const ImgContainer = styled.div`
+  background-image: url(${(props) => props.src});
+  background-position: center;
+  background-size: cover, contain;
+  border-radius: 50%;
+  height: 100px;
+  margin-bottom: 10px;
```

Comment:
I try to use margins as seldom as possible and rather try to manipulate the distance of items with their parent. E.g. grid / gap. But I'm not too sure whether that's an option here.

Reply:
grid gap would affect the two paragraphs, I could wrap them, but them I have to position them again.